pipelinewise/fastsync/commons/tap_mysql.py

```diff
@@ -207,26 +222,32 @@ def get_table_columns(self, table_name):
                             WHEN data_type IN ('blob', 'tinyblob', 'mediumblob', 'longblob')
                                     THEN CONCAT('REPLACE(hex(`', column_name, '`)', ", '\n', ' ')")
                             WHEN data_type IN ('binary', 'varbinary')
-                                    THEN concat('REPLACE(hex(trim(trailing CHAR(0x00) from `',COLUMN_NAME,'`))', ", '\n', ' ')")
+                                    THEN concat('REPLACE(REPLACE(hex(trim(trailing CHAR(0x00) from `',COLUMN_NAME,'`))', ", '\n', ' '), '\r', '')")
```

Comment:
why this is required?

Reply:
the tap was removing newlines included in these fields but we had some data that we were extracting from a Windows system so it included `\r` and `\n`, not only `\n`
From what I see, this is a fix that would apply to more people, not really anything specific to BigQuery